whylog/assistant/regex_assistant/regex_match.py

```diff
@@ -46,8 +47,10 @@ def update_by_regex(self, new_regex):
 
         if new_regex[-1] != '$':
             new_regex += '$'
+        if new_regex[0] != '^':
```

Comment:
this was just removed as a result of application of comment in another PR. Why add it back?